README.md

```diff
@@ -103,15 +103,14 @@ Examples can be found in [optuna/optuna-examples](https://github.com/optuna/optu
 * [XGBoost](https://github.com/optuna/optuna-examples/tree/main/xgboost/xgboost_integration.py)
 
 
-## Web Dashboard (experimental)
+## Web Dashboard
 
-The new Web dashboard is under the development at [optuna-dashboard](https://github.com/optuna/optuna-dashboard).
-It is still experimental, but much better in many regards.
+[Optuna Dashboard](https://github.com/optuna/optuna-dashboard) is a real-time web dashboard for Optuna.
+You can check the optimization history, hyperparameter importances, etc. in graphs and tables.
+You don't need to create a Python script to call [Optuna's visualization](https://optuna.readthedocs.io/en/stable/reference/visualization/index.html) functions.
 Feature requests and bug reports welcome!
```

Comment:
```suggestion
Feature requests and bug reports are welcome!
```
I guess there is a slight grammar error.

Reply:
I'm not sure if it's grammatically correct, but Crissman-san, a native English speaker, actually removed "are".
https://github.com/optuna/optuna/pull/2224#discussion_r559918136

Reply:
Oh..., I see.